src/prefect/settings.py

```diff
@@ -798,6 +798,16 @@ def default_cloud_ui_url(settings, value):
 )
 """The path to a block storage directory to store things in."""
 
+PREFECT_RECORD_STORE_PATH = Setting(
```

Comment:
game to keep this for now, but in general I'm not a fan of settings that only apply to one specific setup; the name makes it seem more applicable than it ends up being in practice

Reply:
What do you think would make this better? Should the name be more specific, or does there need a way to group configuration so that it's clear when they are coordinated?

Reply:
IMO either a generic setting interface that sets the class + config for the class in a universal way, or expecting users to configure this sort of thing on the class itself (which can still use this same path as a default)

Reply:
OK, I'll remove this for now and revisit it when adding a setting to choose the default record store class.